packages/core/angular/src/wallet/wallet.store.ts

```diff
@@ -304,7 +326,9 @@ export class WalletStore extends ComponentStore<WalletState> {
                     return throwError(error);
                 }
 
-                return from(defer(() => adapter.sendTransaction(transaction, connection, options)));
+                return from(defer(() => adapter.sendTransaction(transaction, connection, options))).pipe(
+                    map((txId) => txId as string)
```

Comment:
nit: let's still use `TransactionSignature` here (even though it's just a string)

Reply:
you're right!